Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/UpdateDescription.swift

```diff
@@ -4,16 +4,27 @@ import MongoSwift
  The update description for changed fields in a
  $changeStream operation.
 */
-public class UpdateDescription: Codable {
+public final class UpdateDescription: Codable {
+    enum CodingKeys: CodingKey {
+        case updatedFields, removedFields
+    }
+
     public let updatedFields: Document
     public let removedFields: [String]
 
-    internal init(updatedFields: Document,
-                  removedFields: [String]) {
+    init(updatedFields: Document,
+         removedFields: [String]) {
         self.updatedFields = updatedFields
         self.removedFields = removedFields
     }
 
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        self.updatedFields = try container.decodeIfPresent(Document.self, forKey: .updatedFields) ?? Document()
```

Comment:
[task] might be worth filing a ticket to make android tolerate these fields not being present. We wouldn't want an unintended server change to break the SDK again